test/features/support/get_open_api.js

```diff
@@ -53,16 +58,20 @@ Then('The response should match json schema', () =>
     .to.be.jsonSchema(getOpenApiExpectedSchema)
 );
 
-// Scenario: Retrieve the openAPI description of the specified REST service
+// Scenario Outline: Retrieve the openAPI description of the specified REST service
 Then('The response header content-type should be {string}', expected =>
   specGetOpenAPI.response().to.have.header('content-type', expected)
 );
 
-// Scenario: Unable to retrieve the openAPI description of the specified REST service
+// Scenario Outline: Unable to retrieve the openAPI description of the specified REST service of an invalid path parameter
 Then('The response should have status 400', () =>
   specGetOpenAPI.response().to.have.status(400)
 );
 
-After(() => {
+// Scenario Outline: Unable to retrieve the openAPI description of the specified REST service of an invalid serviceCode parameter
+
+// Scenario: Unable to retrieve the openAPI description of the specified REST service because of missing serviceCode parameter
```

Comment:
There is no code below. If the step what you use here is defined in line 67-69 then better to write this scenario title above it, somehow merge it or write comment below saying where the code is written